sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Latest.java

```diff
@@ -83,7 +116,7 @@ public LatestFn() {}
 
     @Override
     public TimestampedValue<T> addInput(TimestampedValue<T> accumulator,
-        TimestampedValue<T> input) {
+                                        TimestampedValue<T> input) {
```

Comment:
I usually apply google-java-format on the changed bits but in this case I guess I forgot.